Helper/Config.php

```diff
@@ -377,6 +377,7 @@ public function isAllowed(ContextInterface $context)
     /**
      * Init SDK Configuration class for usage
      *
+     * @param int $storeId The store we want to initialize the configuration for
```

Comment:
This param looks like an unused duplicate.

Reply:
Good catch